book/src/suggested-fee-recipient.md

```diff
@@ -168,4 +163,21 @@ curl -X DELETE \
 null
 ```
 
+## Strict Fee Recipient
+
+If the flag `--strict-fee-recipient` is set in the validator client, Lighthouse will refuse to sign any block whose
+`fee_recipient` does not match the `suggested_fee_recipient` sent by this validator. This applies to both the normal
+block proposal flow and block proposals through the builder API. Proposals through the builder API are more likely
+to have a discrepancy in `fee_recipient` so you should be aware of how your connected relay sends proposer payments before
+using this flag. If this flag is used, a fee recipient mismatch in the builder API flow will result in a fallback to the
+local execution engine for payload construction, where a strict fee recipient check will still be applied.
+
+## FAQ
+
+### Why do I have to nominate an Ethereum address as the fee recipient?
+
+You might wonder why the validator can't just accumulate transactions fees in the same way that it
+accumulates other staking rewards. The reason for this is that transaction fees are computed and
+validated by the execution node, and therefore need to be paid to an address that exists on the
+execution chain. In the initial merge fork (Bellatrix) validators have no presence
```

Comment:
Looks like this sentence got cut short?

Reply:
oh yeah good catch, I must have got distracted :P

Reply:
updated now, thanks!